config/__init__.py

```diff
@@ -39,6 +41,8 @@ class Config:
     OPTIONALS = {
         'AWS_LOCAL': 'False',
         'GITHUB_DEFAULT_TEAM_NAME': 'all',
+        'GITHUB_ADMIN_TEAM_NAME': '',
+        'GITHUB_LEADS_TEAM_NAME': '',

```

Comment:
```suggestion
        'GITHUB_ADMIN_TEAM_NAME': 'exec',
        'GITHUB_LEADS_TEAM_NAME': 'leads',

```

Reply:
I think I'd rather leave this without defaults, and have it in our configuration - I still have some distant hope of a generalized Rocket 2 for many orgs 🙃

Reply:
It is in the `sample-env` now for what it's worth